api/build.gradle

```diff
@@ -428,34 +433,38 @@ dependencies {
 
   compile "org.json:json:20160810"
 
-  compile("org.springframework.boot:spring-boot-starter-actuator:$project.ext.SPRING_BOOT_VERSION") {
+  compile("org.apache.tomcat:tomcat-jdbc:9.0.10")
+  compile("org.springframework.boot:spring-boot-starter-tomcat")
+  compile("org.springframework.boot:spring-boot-starter-jdbc")
+  compile("org.springframework.boot:spring-boot-starter-actuator") {
     exclude module: 'spring-boot-starter-tomcat'
     exclude group: 'org.slf4j', module: 'jul-to-slf4j'
   }
 
-  compile("org.springframework.boot:spring-boot-starter-data-jpa:$project.ext.SPRING_BOOT_VERSION") {
+  compile("org.springframework.boot:spring-boot-starter-data-jpa") {
     exclude module: 'spring-boot-starter-tomcat'
     exclude group: 'org.slf4j', module: 'jul-to-slf4j'
   }
 
-  compile("org.springframework.boot:spring-boot-starter-web:$project.ext.SPRING_BOOT_VERSION") {
+  compile("org.springframework.boot:spring-boot-starter-web") {
     exclude module: 'spring-boot-starter-tomcat'
     exclude group: 'org.slf4j', module: 'jul-to-slf4j'
   }
 
-  compile "org.springframework.retry:spring-retry:1.1.5.RELEASE"
-  compile "org.springframework.security:spring-security-core:$project.ext.SPRING_SECURITY_VERSION"
-  compile "org.springframework.security:spring-security-web:$project.ext.SPRING_SECURITY_VERSION"
+  compile "org.springframework.retry:spring-retry:1.3.1"
```

Comment:
Does this one not automatically inherit the version like the other spring packages?

Reply:
oh, right, I installed spring dependency manager now. Giving a shot, that one's version is slightly different than others as the latest version is just 1.3